colorbleed/plugins/maya/publish/collect_renderlayers.py

```diff
@@ -9,7 +9,7 @@
 class CollectMayaRenderlayers(pyblish.api.ContextPlugin):
     """Gather instances by active render layers"""
 
-    order = pyblish.api.CollectorOrder
+    order = pyblish.api.CollectorOrder + 0.01
```

Comment:
Why in particular was this change required?

Reply:
Ah, this is remnant of the previous approach where I first checked if renderlayers were accessible. I will remove it